### PR TITLE
Portuguese translation

### PR DESCRIPTION
**Pull request by geraldolsribeiro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,25 +14,32 @@ This components comes with english, french and german localizations for the noti
...
 1. Add the `hugo-notice` as a submodule to be able to get upstream changes later `git submodule add https://github.com/martignoni/hugo-notice.git themes/hugo-notice`
 2. Add `hugo-notice` as the left-most element of the `theme` list variable in your site's or theme's configuration file `config.yaml` or `config.toml`. Example, with `config.yaml`:
-    ```yaml
-    theme: ["hugo-notice", "my-theme"]
-    ```
-    or, with `config.toml`,
-    ```toml
-    theme = ["hugo-notice", "my-theme"]
-    ```
+
```
Comment (martignoni) [inline README.md]: We should not add blank lines here, because they break the list indentation. Please revert these.

File: i18n/pt.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+- id: warning
```
Comment (martignoni) [inline i18n/pt.yaml]: This is all good. Let's keep it!

**Comment by geraldolsribeiro** [comment]
I drop the last commit (blank lines on README) to keep the history more clean.
Sorry about force-pushed.

**Comment by martignoni** [comment]
Thank you!

**Pull request merged**